packages/reports/addon/components/navi-column-config.ts

```diff
@@ -40,26 +45,36 @@ export default class NaviColumnConfig extends Component<NaviColumnConfigArgs> {
   /**
    * Dimension and metric columns from the request
    */
-  @computed('args.report.request.{columns.[],columns.@each.parameters,filters.[],sorts.[]}')
+  @computed('args.report.request.{columns.[],columns.@each.parameters,filters.[],sorts.[],rollup.columns.[]}')
   get columns(): ConfigColumn[] {
     const { request } = this.args.report;
     const requiredColumns = this.requestConstrainer.getConstrainedProperties(request).columns || new Set();
     if (request.table !== undefined) {
-      const { columns, filters } = request;
+      const { columns, filters, rollup } = request;
 
       const filteredColumns = filters.map(({ canonicalName }) => canonicalName);
 
       return columns.map((column) => {
         return {
           isFiltered: filteredColumns.includes(column.canonicalName),
           isRequired: requiredColumns.has(column),
+          isRollup: rollup.columns.includes(column.cid),
           fragment: column,
         };
       });
     }
     return [];
   }
 
+  @computed('args.report.request.dataSource')
+  get supportsSubtotal(): boolean {
+    if (!this.args.report.request.dataSource) {
+      return false;
+    }
```

Comment:
```suggestion
```
Was this for request v1? v2 requires dataSource

https://github.com/yavin-dev/framework/blob/8f7e88ef86248513080274cec9085137bc69e92f/packages/core/addon/models/bard-request-v2/request.ts#L93

Reply:
Fixed, I think I needed it before for previous code, didn't realized I coded myself out of needing this.